sweddit_combined/localisation/em_modifers_l_english.yml

```diff
@@ -1,4 +1,4 @@
-﻿l_english:
+l_english:
```

Comment:
This file shouldn't have been pushed.

Probably best to remove all of these and squash merge the PR.